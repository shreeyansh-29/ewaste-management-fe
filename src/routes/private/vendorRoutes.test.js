/* eslint-disable no-undef */
import React from "react";
import Enzyme, {shallow} from "enzyme";
import Adatper from "@wojtekmaj/enzyme-adapter-react-17";
import toJson from "enzyme-to-json";
import VendorRoutes from "./vendorRoutes";

Enzyme.configure({adapter: new Adatper()});

describe("testing Vendor Routes", () => {
  const wrapper = shallow(<VendorRoutes />);

  it("test vendorRoutes", () => {
    expect.assertions(1);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
