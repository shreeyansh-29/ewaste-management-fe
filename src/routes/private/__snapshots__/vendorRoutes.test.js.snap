// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Vendor Routes test vendorRoutes 1`] = `
<Routes>
  <Route
    element={<VendorHome />}
    exact={true}
    path="/VendorHome"
  />
  <Route
    element={<Memo(Connect(VendorNav)) />}
    exact={true}
    path="/VendorNav"
  />
  <Route
    element={<Memo(Connect(VendorProfile)) />}
    exact={true}
    path="/VendorProfile"
  />
  <Route
    element={<Memo(Connect(Sales)) />}
    exact={true}
    path="/MyOrders/PurchaseItems"
  />
  <Route
    element={<Memo(Connect(MyOrders)) />}
    exact={true}
    path="/MyOrders/SalesSummary"
  />
  <Route
    element={<AuthProtected />}
    path="/"
  />
  <Route
    element={<PageNotFound />}
    exact={true}
    path="*"
  />
</Routes>
`;
